test_files/declare/declare-module.d.ts

```diff
@@ -3,8 +3,10 @@
 // "declare global" should show up in the global namespace.
 declare global {
   var globalX: string;
-  namespace subnamespace {
+  export class GlobalClass {}
+  export namespace subnamespace {
```

Comment:
This is just an ordinary namespace, not a sub namespace right?
Maybe just global in its name so it's more obvious in the js?